lib/tabula.js

```diff
@@ -287,7 +289,10 @@ function tabulaFormat(items, options) {
             if (options.dottedLookup) {
                 try {
                     cell = dottedLookup(o, col.lookup);
-                } catch (e) {}
+                } catch (e) {
+                    if (options.caseInsensitiveLookup)
+                        throw new Error(e);
```

Comment:
since `e` is already an error, you shouldn't need the `new Error` constructor, ie.

``` js
if (options.caseInsensitiveLookup)
  throw e;
```